content/webapp/components/SearchNoResults/index.tsx

```diff
@@ -18,6 +18,11 @@ const SearchNoResults: FunctionComponent<Props> = ({
   query,
   hasFilters,
 }: Props) => {
+  useEffect(() => {
+    window.dataLayer?.push({
+      event: 'noSearchResults',
+    });
+  }, []);
```

Comment:
It fired twice, is that an issue?

Reply:
<img width="965" alt="Screenshot 2025-05-29 at 09 26 50" src="https://github.com/user-attachments/assets/adacae7e-c208-47f8-87eb-c58c7c9cbbfb" />
<img width="892" alt="Screenshot 2025-05-29 at 09 26 53" src="https://github.com/user-attachments/assets/8415f599-a041-485d-a312-da99163054f0" />

Reply:
Ohhh ok nice! Yes can just double check once in staging 👍

Reply:
<img width="286" alt="image" src="https://github.com/user-attachments/assets/d7c8e6fc-0449-4002-811a-f506c7d7340e" />